src/cleanfigure.m

```diff
@@ -206,8 +203,9 @@ function pruneOutsideBox(meta, handle)
   %hasMarkers = ~strcmp(marker,'none');
   hasLines = true;
```

Comment:
I traced it back to ce899382ce6c0edfc69bc415cf0b769dce0f575f which is almost 2 years old. The commit message suggests it was copied in from the main matlab2tikz code (back when `cleanfigure` still was a part of it), but I can't find the corresponding part in that file right now.

For this case, I'd say: just put a TODO above those lines:
```matlab
%TODO: test again with lines uncommented (see #787 and ce899382)
```